main.js

```diff
@@ -41,15 +41,22 @@ startActivityButton.addEventListener('click', submitForm);
 startTimerButton.addEventListener('click', function() {
   currentActivity.startTimer();
 });
-logActivityButton.addEventListener('click', logActivity);
+logActivityButton.addEventListener('click', logActivtyEvents);
 
-function logActivity() {
-  console.log('yes');
+//Add functions below
+function completeActivity() {
+  countdownClock.innerText = "Congrats! Keep it up!!";
+  countdownClock.style.fontSize = "3em";
+  startTimerButton.innerText = "COMPLETE";
+  show(logActivityButton);
+  currentActivity.markComplete();
+}
+
+function logActivtyEvents() {
   hide(timerBoxWrapper);
   show(createNewActivityButton);
 }
 
-//Add functions below
```

Comment:
Was this line removed on purpose? I don't think we need it necessarily but we should be consistent. Right now we do have a few comment lines to organize our JS code